edgedb/protocol/codecs/object.pyx

```diff
@@ -21,7 +21,74 @@
 cdef class ObjectCodec(BaseNamedRecordCodec):
 
     cdef encode(self, WriteBuffer buf, object obj):
```

Comment:
Let's name this `encode_args` as the method is specific to argument encoding.

Reply:
changed